processing/audio_analysis.py

```diff
@@ -47,12 +60,46 @@ def process(recording, jwtKey, conf):
         input_filename = temp_path / ("recording" + input_extension)
         logger.debug("downloading recording to %s", input_filename)
         api.download_file(jwtKey, str(input_filename))
+        analysis = analyse(input_filename, conf)
+        new_metadata = {}
+
+        if analysis["species_identify"]:
```

Comment:
should these be analysis.get("species_identify") so it will return none if it doesnt exist?